pkg/controllers/ocmcontroller/ocmagent.go

```diff
@@ -0,0 +1,119 @@
+// Copyright Contributors to the Open Cluster Management project
+package ocmcontroller
+
+import (
+	"context"
+	"os"
+	"path"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	aggregatorapiserver "k8s.io/kube-aggregator/pkg/apiserver"
+
+	clusterclient "open-cluster-management.io/api/client/cluster/clientset/versioned"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+	"open-cluster-management.io/multicluster-controlplane/pkg/agent"
+)
+
+const (
+	defaultOCMAgentNamespace = "open-cluster-management-agent"
+	selfManagedClusterName   = "local-cluster"
```

Comment:
Do we have those constants exposed in open-cluster-management.io/api? if so, we may reference there.

Reply:
maybe we can put the `defaultOCMAgentNamespace` to the api repo, I will add a todo here firstly :)